### PR TITLE
chore(release): 1.2.1

### PR DESCRIPTION
**Pull request by Matthew-Mallimo**
Adds a fix for holocron to prevent bad modules from being put in the module map

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,18 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [1.2.1](https://github.com/americanexpress/holocron/compare/v1.1.5...v1.2.1) (2022-02-10)
+
+
+### Bug Fixes
+
+* **externals:** mismatched externals caused one-app to fail to start ([#90](https://github.com/americanexpress/holocron/issues/90)) ([fe49f8c](https://github.com/americanexpress/holocron/commit/fe49f8cf82d7e00dd9fac925cd61d23240d92469))
```
Comment (Matthew-Mallimo) [inline CHANGELOG.md:11]: im not sure why this was included in the release notes

Comment (10xLaCroixDrinker) [reply CHANGELOG.md:11]: Was the previous tag not on `main`?

Comment (10xLaCroixDrinker) [reply CHANGELOG.md:11]: Looks like it wasn't. That would cause this

Comment (JAdshead) [reply CHANGELOG.md:11]: when merging we need to make sure that we rebase and merge so that the tagged commit ends up on main

Comment (JAdshead) [inline CHANGELOG.md:11]:
```suggestion
```

File: packages/holocron/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,18 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [1.2.1](https://github.com/americanexpress/holocron/compare/v1.1.5...v1.2.1) (2022-02-10)
+
+
+### Bug Fixes
+
+* **externals:** mismatched externals caused one-app to fail to start ([#90](https://github.com/americanexpress/holocron/issues/90)) ([fe49f8c](https://github.com/americanexpress/holocron/commit/fe49f8cf82d7e00dd9fac925cd61d23240d92469))
```
Comment (JAdshead) [inline packages/holocron/CHANGELOG.md:11]:
```suggestion
```

**Review by 10xLaCroixDrinker — APPROVED** [review]

**Review by JAdshead — APPROVED** [review]

**Pull request merged**